.github/workflows/tests.yml

```diff
@@ -80,7 +80,10 @@ jobs:
     - name: Install pyyaml
       if: matrix.os == 'ubuntu-20.04'
       run: sudo apt-get install -y libyaml-dev
+    # save workspaces to speed up testing
+    - name: set CAPA_SAVE_WORKSPACE environment variable
+      run: echo "CAPA_SAVE_WORKSPACE=True" >> $GITHUB_ENV
```

Comment:
I hope this speed things up again

Reply:
confirmed that this works in https://github.com/fireeye/capa/runs/2941515558
`DEBUG:capa:saving workspace`